test/integration/testdata/dummytype/alpha.go

```diff
@@ -0,0 +1,27 @@
+package dummytype
```

Comment:
Do we need the new package? Maybe I'd have put everything together

Reply:
I'd like to test the sorting behavior on package level, e.g.
- `-i "./testdata/listtype,./testdata/dummytype"`
- `-i "./testdata/dummytype,./testdata/listtype"`

The violations from `dummytype` should come first in report.

It turns out that `k8s.io/gen` handles the package-level sorting when walking through the package, but I think it's still good to test the behavior.

Reply:
Good point! Maybe add a comment? Or maybe it's just me ...

Reply:
Will do